libs/ui/elements/src/lib/metadata-info/metadata-info.component.ts

```diff
@@ -73,6 +73,37 @@ export class MetadataInfoComponent {
     }
   }
 
+  get temporalExtent(): string {
```

Comment:
This should probably not be a string

Reply:
Also two more cases need to be handled: when there is only a start date, and only an end date